app/src/main/java/com/example/habitsmasher/ui/dashboard/HabitListFragment.java

```diff
@@ -48,6 +49,8 @@ public View onCreateView(@NonNull LayoutInflater inflater,
                              ViewGroup container, Bundle savedInstanceState) {
         Context context = getContext();
 
+        ((AppCompatActivity) requireActivity()).getSupportActionBar().setTitle("Habit List");
```

Comment:
I'm assuming this sets the title of the screen back to what it should be (habit list)?

Reply:
Yep, fixes the bug that was causing our UI tests to fail haha.